src/libcollectionstest/vec.rs

```diff
@@ -213,6 +213,16 @@ fn test_retain() {
 }
 
 #[test]
+fn test_retain_mut() {
```

Comment:
This was following existing convention; I'm not sure of the exact benefits of having the test in both places though.